ang/test/civiawards/award-creation/directives/review-panels/review-panels.directive.spec.js

```diff
@@ -194,6 +194,54 @@
           }, jasmine.any(Object));
         });
       });
+
+      describe('delete button', () => {
+        describe('when creating new review panel', () => {
+          it('hides the delete button inside the popup', () => {
```

Comment:
The it says that it checks if the button is hidden, but logic checks if save button is present. 
I suggest we change the `it` statement to say 
`it('has only the save action button inside popup')`
and inside another it we should have `has both save and delete action button inside the popup`.